gooey/gui/components/menubar.py

```diff
@@ -78,4 +80,23 @@ def spawnAboutDialog(self, item, *args, **kwargs):
             if field in item:
                 getattr(about, method)(item[field])
 
-        three_to_four.AboutBox(about)
\ No newline at end of file
+        three_to_four.AboutBox(about)
+
+    def spawnPrintDialog(self, item, *args, **kwargs):
+        fonts = {

```

Comment:
I lean towards just using whatever is set via the `terminal_font_xxx` family of options (see [here](https://github.com/chriskiehl/Gooey/blob/master/gooey/python_bindings/config_generator.py#L90)), rather than introducing another set of font options just for printing. I'd wager that whatever font they choose for the terminal, will be the one they want to see printed out. 

Lemme know if you have an alternative use case in mind that warrants the extra font controls, though.